package.json

```diff
@@ -178,13 +178,11 @@
   },
   "lint-staged": {
     "*.{js,cjs,mjs}": [
-      "eslint --cache --fix"
+      "eslint --fix"
```

Comment:
Why do you remove `--cache`? Is it to do nothing?

Reply:
But why we should have broken cache? I think it should be a problem with eslint and we will catch this on CI